g3w-admin/qdjango/api/projects/serializers.py

```diff
@@ -930,73 +927,63 @@ def __init__(self, instance=None, data=empty, **kwargs):
                 'results': body['results'],
-                'filter': [],
                 'dozoomtoextent': body['dozoomtoextent'],
-                # 'zoom': body['zoom']
-            }
-
-            # other layers
-            if 'otherlayers' in body:
-                ret['options'].update({
-                    'otherquerylayerids': body['otherlayers']
-                })
-
-            for field in body['fields']:
-
-                # if widgettype is selectbox, get values
-                if 'widgettype' in field and field['widgettype'] == 'selectbox':
-
-                    field['input']['type'] = 'selectfield'
-                    field['input']['options']['values'] = []
-
-                    edittype = edittypes[field['name']]
-
-                    # check if field has a widget edit type
-                    widget_type = edittype['widgetv2type']
-                    if field['name'] in edittypes and widget_type in ('ValueMap', 'ValueRelation'):
-                        if widget_type == 'ValueMap':
-                            field['input']['options']['values'] = edittype['values']
-                        else:
-
-                            # Add layer params
-                            field['input']['options']['key'] = edittype['Value']
-                            field['input']['options']['value'] = edittype['Key']
-                            field['input']['options']['layer_id'] = edittype['Layer']
-
-                # For AutoccOmpleteBox input type
-                if 'widgettype' in field and field['widgettype'] == 'autocompletebox':
-                    field['input']['type'] = 'autocompletefield'
-
-                input = field['input']
-                input['options']['blanktext'] = field['blanktext']
-                ret['options']['filter'].append({
+                # other layers
+                'otherquerylayerids': body.get('otherlayers', []), 
+                # search inputs
+                'filter': [{
                     'op': field['filterop'],
                     'attribute': field['name'],
                     'label': field['label'],
-                    'input': input,
-                    'logicop': field.get('logicop', 'AND').upper()
-                })
-
-            # rewrite type if relations is set relations
-            if 'relations' in body and body['relations'] != '':
-                ret['type'] = 'search_1n'
-                ret['options']['search_1n_relationid'] = body['relations']
-                del(body['relations'])
+                    'logicop': field.get('logicop', 'AND').upper(),
+                    'input': {
+                        **field['input'],
+                        'widget_type': etype(field, 'widgetv2type'),
+                        'type': ({
+                            'autocompletebox': 'autocompletefield',
+                            'selectbox': 'selectfield'
+                        }).get(field.get('widgettype'), field['input'].get('type')),
```

Comment:
@wlorenzetti the `numberfield` type should also be returned in here (currently not implement in v3.7):


## OK: "Volume greater than" appears as number field

![image](https://github.com/g3w-suite/g3w-admin/assets/9614886/d7840158-c5e6-4781-8f08-345e40471cc2)

## KO: "Volume greater than" appears as text field

![image](https://github.com/g3w-suite/g3w-admin/assets/9614886/39a8e6dc-2fa1-4485-ac7b-66baeb4fb6dc)

## Sample config: "Type and volume"

![image](https://github.com/g3w-suite/g3w-admin/assets/9614886/b3885ab6-b6a3-4020-8877-eb98ad09e780)